services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/ScanEnhancedRequest.java

```diff
@@ -17,11 +17,17 @@
 
 import java.util.HashMap;
 import java.util.Map;
-
 import software.amazon.awssdk.annotations.SdkPublicApi;
+import software.amazon.awssdk.enhanced.dynamodb.DynamoDbTable;
 import software.amazon.awssdk.enhanced.dynamodb.Expression;
 import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
 
+/**
+ * Defines parameters used to when querying a DynamoDb table or index using the query() operation (such as
+ * {@link DynamoDbTable#scan(ScanEnhancedRequest)}).
```

Comment:
Replace query with scan. Actually search this whole class for 'query', I see it popping up all over the place.

Reply:
Thanks for finding them. Fixing.